package.json

```diff
@@ -26,6 +26,16 @@
       "default": "./lib/universal.js"
     }
   },
+  "typesVersions": {
+    "*": {
+      "use-at-your-own-risk": [
+        "./lib/types/use-at-your-own-risk.d.ts"
+      ],
+      "rules": [
+        "./lib/types/rules/index.d.ts"
+      ]
+    }
```

Comment:
Shall we also include `"universal"` in this section, or is it intentionally omitted for some reason?